02/MirrorString.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include<string>
+#include "MirrorString.h"
+
+using namespace std;
+
+namespace samples
+{
+	void MirrorStringExample()
+	{
+		string line = "Hello World!";
+
+		for (size_t i = line.length(); i--> 0;)
```

Comment:
I have a question about this code.. 
I tried to compile this code after modifying as you commented, but "string subscript out of range" error occurred.
I think it's because variable i had been -1.
Is this code right?